doc/ref_arch/openstack/chapters/chapter01.rst

```diff
@@ -103,11 +103,39 @@ Wallaby."
 Principles
 ----------
 
-OpenStack Reference Architecture must obey to the following set of
-principles described in:
+Architectural principles
+~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
I have opened an Issue as there are grammatical and other errors in the Architectural Principles